OpenRA.Mods.Common/Traits/Sound/AmbientSound.cs

```diff
@@ -0,0 +1,31 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Play a looping audio.")]
```

Comment:
`audio` needs to become `sounds` or this needs to be reworded a bit (will try and come up with something).
The English here is off.